### PR TITLE
Update spp image

### PR DESCRIPTION
**Pull request by masafumimori**
- update spp image
- fix image size adjustment
- change the calculation for plot position

Note:
Apparently, as the image gets smaller, the thickness changes which requires different calculations for positioning larger image plots.
For now I just distinguish for spp and the others, but we might need more sophisticated calculation methods in the future as there will be many different size images for plots.
The image shows how it looks like now.
Screenshot 2022-08-22 at 9 52 03

### CODE REVIEW

File: components/map/map-helper.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import Plot, { Position2D } from './plots/plot'
-// TODO: should change image once design for SPP is ready
-import SPP from 'public/images/map-tile-strip-lake.png'
```
Comment (lorenzopolidori) [inline components/map/map-helper.ts:3]: We should also remove `public/images/map-tile-strip-lake.png`.

**Comment by lorenzopolidori** [comment]
There seems to be an issue with positioning the image. If I resize the browser window and I make it vertically shorter, the SPP moves downwards as in the video below.

https://user-images.githubusercontent.com/2437709/186008788-d5a6afd2-8682-4dea-87e7-514064a3d3a5.mov

**Comment by masafumimori** [comment]
I fixed tasks below as well

- https://trello.com/c/Z8IxKUJh/36-make-it-possible-to-show-terraland-nfts-with-new-illustrations
- https://trello.com/c/fTjxUWJO/3-implement-basic-map-functionality (`Remove MAGIC_NUMBER_TO_ADJUST`)

**Comment by lorenzopolidori** [comment]
> Also, just so you know, the thickness of SPP image looks different from others, which I suppose we need an offset adjustment for that specific image.

Thanks, I have seen that. I asked our illustrator to fix the thickness of the SPP surface layer.

**Review by lorenzopolidori — APPROVED** [review]

**Pull request merged**